src/v2/guide/migration.md

```diff
@@ -1540,47 +1540,47 @@ new Vue({
 
 ## Global Config
 
-### `Vue.config.debug` <sup>removed</sup>
+### `Vue.config.debug` <sup>removido</sup>
 
-No longer necessary, since warnings come with stack traces by default now.
+Ya no es necesario, ya que ahora las advertencias vienen con seguimiento de pila por defecto.
```

Comment:
I think we should leave `stack trace` instead of translating it to `seguimiento de pila`. What do you think?